code/week5/week5.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
Is there a reason these includes are in a separate file and not just at the top of main.cpp?